test/test_images/consumer-group-lag-provider-test/admin.go

```diff
@@ -0,0 +1,212 @@
+/*
+ * Copyright 2021 The Knative Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package main
+
+import (
+	"context"
+	"log"
+	"time"
+
+	"github.com/Shopify/sarama"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	"knative.dev/eventing-kafka-broker/control-plane/pkg/kafka"
+	testingpkg "knative.dev/eventing-kafka-broker/test/pkg/testing"
+)
+
+func main() {
```

Comment:
Is this thing testing your test code? If that's the case, do we really need it? Your code to check the consumer group lag is quite easy, unit test should be enough i guess?

Reply:
I prefer to keep this test, I discovered a bug with this test :upside_down_face: which is a signal that it's useful.
I was debugging the data plane but the bug was actually in the test code.

Reply:
I see... TBH, an integration test for the test code is weird :smile: Maybe can you just comment that this is useful for testing the test code and maybe disable it in our test pipeline?

Reply:
Disabled.